src/components/SearchBar/SearchBar.js

```diff
@@ -19,23 +16,22 @@ class SearchBar extends React.Component {
       sortBy: 'best_match'
 
     };
-  this.handleTermChange = this.handleTermChange.bind(this);
-  this.handleLocationChange = this.handleLocationChange.bind(this);
-  this.handleSearch = this.handleSearch.bind(this);
+    this.handleTermChange = this.handleTermChange.bind(this);
+    this.handleLocationChange = this.handleLocationChange.bind(this);
+    this.handleSearch = this.handleSearch.bind(this);
 
   }
   renderSortByOptions() {
     return Object.keys(sortByOptions).map(sortByOption => {
-      let sortByOptionsValue = sortByOptions[sortByOption];
+      const sortByOptionsValue = sortByOptions[sortByOption];
```

Comment:
Made this const as the value is not changing - const optimises more highly when the code compiles.